CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+# 3.3.0
+
+This release improves how collateral plugins price LP tokens and moves reward claiming out to the asset plugin level.
+
+## Upgrade Steps
+
+Swapout all collateral plugins with appreciation.
+
+All collateral plugins should be upgraded. Only compound-v2 ERC20 needs to be rotated, and it is purely optional to provide gas savings.
+
+### Core Protocol Contracts
+
+- `BackingManager` + `RevenueTrader`
+  - Change `claimRewards()` to delegatecall to the list of registered plugins
+
+## Plugins
+
+### Assets
+
+- compound-v2
+  - Remove `CTokenWrapper`
+  - Add reward claiming logic to `claimRewards()`
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- compound-v3
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- curve
+  - Make `price()` more resistant to manipulation by MEV
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- convex
+  - Make `price()` more resistant to manipulation by MEV
+  - Emit `RewardsClaimed` event during `claimRewards()`
+- frax-eth
+  - Emit `RewardsClaimed` event during `claimRewards()`
```

Comment:
i dont see this in this pr

Reply:
oooo I think a merge went wrong

Reply:
Nvm, there's no rewards for `frax-eth` or `yearn-v2`. Amended